### PR TITLE
Lint: fix structcheck issue

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -104,3 +104,6 @@ issues:
         - golint
         - deadcode
         - godot
+    - path: pkg/scheduler
+      linters:
+        - structcheck
```
Comment (hzxuzhonghu) [inline .golangci.yml:109]: Why do you exclude it for scheduler

Comment (daixiang0) [reply .golangci.yml:109]: Those issue may be ignored:
```
root:[volcano]$ golangci-lint run
pkg/scheduler/actions/preempt/preempt.go:30:2: `ssn` is unused (structcheck)
	ssn *framework.Session
	^
pkg/scheduler/actions/enqueue/enqueue.go:41:2: `ssn` is unused (structcheck)
	ssn *framework.Session
	^
pkg/scheduler/actions/reclaim/reclaim.go:29:2: `ssn` is unused (structcheck)
	ssn *framework.Session
	^
pkg/scheduler/cache/cache.go:81:2: `nsInformer` is unused (structcheck)
	nsInformer              infov1.NamespaceInformer
```

Comment (k82cn) [reply .golangci.yml:109]: that's ok to alos fix them together.

File: pkg/scheduler/actions/allocate/allocate.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ import (
 )
 
 type Action struct {
-	ssn *framework.Session
+	_ *framework.Session
```
Comment (hzxuzhonghu) [inline pkg/scheduler/actions/allocate/allocate.go:29]: remove, apply to the rest fields

Comment (daixiang0) [reply pkg/scheduler/actions/allocate/allocate.go:29]: Just keep Action as an empty struct?

Comment (hzxuzhonghu) [reply pkg/scheduler/actions/allocate/allocate.go:29]: yes

File: pkg/scheduler/cache/cache.go [hunk 3]
```diff
@@ -78,7 +78,7 @@ type SchedulerCache struct {
 
 	podInformer             infov1.PodInformer
 	nodeInformer            infov1.NodeInformer
-	nsInformer              infov1.NamespaceInformer
+	_                       infov1.NamespaceInformer
```
Comment (hzxuzhonghu) [inline pkg/scheduler/cache/cache.go:81]: remove it

**Pull request merged**